arches/app/models/models.py

```diff
@@ -1183,8 +1183,7 @@ class SearchComponent(models.Model):
     type = models.TextField()
     componentpath = models.TextField(unique=True)
     componentname = models.TextField(unique=True)
```

Comment:
remove the `unique` conststraint
```suggestion
    componentpath = models.TextField()
```

Reply:
fix in latest

Reply:
It looks like this has been changed back to unique?

Reply:
Yea I reverted the change. I was seeing a weird issue preventing the migration from working, regarding the index of the search_components table (which was generated automatically I think) not being refreshed correctly. That got me thinking more about the change, and then I realized that since componentnames are unique and each componentpath is 1:1 with its component, may as well keep componentpath unique to avoid unintended errors.